reference/src/client/include/sa_key.h

```diff
@@ -93,6 +93,112 @@ sa_status sa_key_export(
         size_t mixin_length,
         sa_key key);
 
+/* **TA Key Type Definition**
+
+The operator provisioning key type will be communicated through the following enumeration:
+*/
+typedef enum  {
+  WIDEVINE_OEM_PROVISIONING,
+  PLAYREADY_MODEL_PROVISIONING,
+  NETFLIX_PROVISIONING
+} sa_ta_key_type;
```

Comment:
Other types and enumeration definitions are in sa_types.h and sa_key.h contains function declarations for key module. Could we keep this concept for TA key provision?